db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_08_31_172741) do
+ActiveRecord::Schema.define(version: 2020_04_29_005140) do
```

Comment:
please update this number. we deployed some migrations.

Reply:
Updated on branch.

Reply:
Merge conflict has been resolved.